MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+graft libcomps*
```

Comment:
Is adding this file intentional?

Reply:
Yes, it ensures that source packages can be built properly, because otherwise not all the necessary files are included by default.  Granted that source packages are only a "backup" if for some reason there isn't a wheel available that is compatible with your platform.

There are attempts in the Python ecosystem to get rid of all the separate configuration files that are needed (setup.py, manifest.in, etc.) and standardize on pyproject.toml, but it will be a while until they can be reliably avoided.

Reply:
That's a bit annoying, I don't have thorough knowledge of the python build system, but since this only needs to be added now, I assume it wasn't needed until now and thus is used only in this special case. That makes it quite hard to maintain for someone who doesn't know all the details.